src/python/serenity_types/pricing/derivatives/options/valuation.py

```diff
@@ -182,27 +182,17 @@ class OptionValuationRequest(BaseModel):
     nor their own yield curve for any given asset, the system will load the default as-of the as_of_time.
     """
 
+    vol_model: VolModel
+    """
+    The volatility model used for evaluation purposes
+    """
```

Comment:
Should this be Optional? If either UUID or InterpolatedVolatilitySurface is provided, then this parameter needs to be consistent with the InterpolatedVolatilitySurface.definition.vol_model, so this maybe makes it needlessly brittle if it must be specified.

Reply:
(I do understand why you might want to provide it in case the system is resolving surfaces for you.)

Reply:
Yes it should be optional.  If no vol surface id or interpolated vol surface is provided by the use, the system has to decide which vol surface to load (Black Scholes or SVI)

Reply:
Yep, and it would make sense for the model config to specify what is the default VolModel in such cases. Kindly push this change together with the PR; I will approve now.